### PR TITLE
Do not look at this pr, do not speak of this pr, do not think of this pr

### PR DESCRIPTION
**Pull request by MrsTonedOne**
I'm testing rabbits

### CODE REVIEW

**Comment by Kylerace** [comment]
:trollface:

**Comment by MrDoomBringer** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title that explains the purpose of the PR.

Your current title does not meet requirements, so please update it.

**Comment by celotajstg** [comment]
I have looked at this PR.
I have seen this PR.

I am now curious if it leads to improvements and any issues.

**Comment by interestingusernam3** [comment]
eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

File: code/__HELPERS/_lists.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 #define LAZYADDASSOC(L, K, V) if(!L) { L = list(); } L[K] += V;
 ///This is used to add onto lazy assoc list when the value you're adding is a /list/. This one has extra safety over lazyaddassoc because the value could be null (and thus cant be used to += objects)
 #define LAZYADDASSOCLIST(L, K, V) if(!L) { L = list(); } L[K] += list(V);
-#define LAZYREMOVEASSOC(L, K, V) if(L) { if(L[K]) { L[K] -= V; if(!length(L[K])) L -= K; } if(!length(L)) L = null; }
+#define LAZYREMOVEASSOC(L, K, V) if(L) { if(L[K]) { L[K] -= V; if(!length(L[K])) L -= K; } /*if(!length(L)) L = null; */}
 #define LAZYACCESSASSOC(L, I, K) L ? L[I] ? L[I][K] ? L[I][K] : null : null : null
 #define QDEL_LAZYLIST(L) for(var/I in L) qdel(I); L = null;
```
Comment (Arokha) [inline code/__HELPERS/_lists.dm:30]: I'm ignoring the title!

This one also sets the list var to null

**Comment by JJRcop** [comment]
1000px-SgtShultz

**Pull request closed without merging**